examples/nwc/wallet-service/notifications.ts

```
@@ -0,0 +1,108 @@
+const rl = readline.createInterface({ input, output });
+
+const relayUrls = (
+  (await rl.question(
+    `Relay URLs, comma separated (${DEFAULT_RELAY_URLs}): `,
+  )) || DEFAULT_RELAY_URLs
+).split(",");
+rl.close();
+
+const nwcUrl = `nostr+walletconnect://${walletServicePubkey}?relay=${relayUrls.join("&relay=")}&secret=${clientSecretKey}`;
+
+console.info("enter this NWC URL in a client: ", nwcUrl);
```

ESLint

[error] Unexpected console statement.

(no-console)